arcgisUM.py

```diff
@@ -84,7 +84,7 @@ def getArcGISGroupByTitle(arcGISAdmin, title):
     :return: ArcGIS Group object or None
     :rtype: Group or None
     """
-    searchString = "title:"+title
+    searchString = f'title:"{title}"'
```

Comment:
Do you think we should escape the character `"` within `title`? It didn't seem to matter in testing, but it didn't either when I added something like this:

```
    escapedTitle = title.translate(str.maketrans({ '"':  r'\"' }))

    searchString = f'title:{escapedTitle}'
```

Reply:
checked in the `"` escape as suggested above.